src/config.js

```diff
@@ -48,6 +48,19 @@ export const urls = {
   errorReporting:
     env['errorReportingUrl'] || 'https://amp-error-reporting.appspot.com/r',
   localDev: env['localDev'] || false,
+  /**
+   * These domains are trusted with more sensitive viewer operations such as
+   * propagating the referrer. If you believe your domain should be here,
+   * file the issue on GitHub to discuss. The process will be similar
+   * (but somewhat more stringent) to the one described in the [3p/README.md](
+   * https://github.com/ampproject/amphtml/blob/master/3p/README.md)
+   *
+   * @export {!Array<!RegExp>}
+   */
+  trustedViewerHosts: [
+    /(^|\.)google\.(com?|[a-z]{2}|com?\.[a-z]{2}|cat)$/,
+    /(^|\.)gmail\.dev$/,
```

Comment:
Why not just add this to the other regex?

Reply:
I also thought it was more readable as a separate regex. Please change back. 😄